### ISSUE TITLE
Microsoft comments on Graphics Contrast

### ISSUE THREAD

**Issue by alexswli** [w3c/wcag21#181]
Clarity of text:

a. The term "essential for understanding" is in the SC text yet there is an "essential exception on presentation".  This may make it hard to parse out when the exception applies.

b. I assume the graphic is no longer essential for understanding if the content is explained textually elsewhere.  Please make that clearer if possible.

Implementable:

a. There are other graphics, such as country flags, that ought to be exempted. 

b.More complex graphics where shades and gradient presents uncertainty as to whether they meet the requirements and how testing can be done.  Please consider more complex graphics in this SC.

**Comment by alastc** [comment]
Hi Alex,

Clarity a) I think we are moving to "required for understanding", so it won't clash with the 'essential' exception.

Clarity b) The [understanding doc](rawgit.com/w3c/wcag21/graphics-contrast/understanding/21/graphics-contrast.html) has that high up in the page, is that ok?

Implementable a) I would have thought flags would be excepted by both 'logotypes' and 'essential', do you think others will disagree? I can add that as an example in the understanding. 

Implementable b) Mike also asked, I went through an example in #435, if that makes sense to you both I'll add it to the understanding doc.

**Comment by awkawk** [comment]
Thank you for the comments. We will add examples as suggested to the understanding docs. Your points "a" and "b" are addressed with the latest version. Regarding complex graphics and graphics with gradients, we believe that the latest language helps ease the challenges and will look for additional feedback from Microsoft and others on this point.
